common/fixation_migrations.go

```diff
@@ -0,0 +1,77 @@
+package common
+
+import (
+	"fmt"
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/common/types"
+)
+
+func prefixForErrors(from uint64) string {
+	return fmt.Sprintf("FixationStore: migration from version %d", from)
+}
+
+func (fs *FixationStore) MigrateVersion(ctx sdk.Context) (err error) {
```

Comment:
As we discussed, for FixationStore (and later TimerStore) we keep it as is, because (1) they are not independent modules, (2) to avoid exposing internal API (or copying existing code). This decision can be revisited later.

Reply:
As for "x/plans" and "x/projects", their upgrade involves just calling FixationStore's MigrateVersion(), not necessary to create a separate directory + function for that. See updated PR.